build/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:alpine as builder
+
+RUN apk add gcc libc-dev
+# Create initial dir
+RUN mkdir archon
+COPY ./ ./archon
+WORKDIR archon
+# Compile the code
+RUN mkdir .bin
+ENV GOBIN=/go/archon/.bin
+RUN go install ./cmd/*
+# Create a directory for the server files
+RUN mkdir archon_server
+WORKDIR archon_server
+RUN cp ../.bin/* .
+# Copy the supporting files
+RUN cp -r ../setup/* .
+# Overwrite config
+COPY /build/docker_config.yaml ./config.yaml
+# Generate certificate
+RUN ./generate_cert -ip 0.0.0.0/32
+# Create test user account
+FROM builder as account
+# !!! This requires an existing postgres connection !!!
+CMD ["./account", "-config", "./config.yaml", "-username", "testuser", "-password", "testpass", "-email", "test@mail", "add"]
```

Comment:
Can you make the config file an arg that defaults to `/.config.yaml`? The reason I ask is that while I strive to keep the config file in `setup/` up to date I do have a custom one I develop with that's useful.

Reply:
Removed - config overriden